darts/metrics/metrics.py

```diff
@@ -33,10 +55,14 @@ def _get_values_or_raise(series_a: TimeSeries,
                                                                     series_a.time_index(), series_b.time_index()),
                  logger)
 
-    return series_a_common.values(), series_b_common.values()
+    return series_a_common.univariate_values(), series_b_common.univariate_values()
 
 
-def mae(series1: TimeSeries, series2: TimeSeries, intersect: bool = True) -> float:
+@multivariate_support
+def mae(series1: TimeSeries,
+        series2: TimeSeries,
+        intersect: bool = True,
+        reduction: Callable[[np.ndarray], float] = np.mean) -> float:
```

Comment:
why is the type `[np.ndarray]` in brackets here?

Reply:
I believe this is the proper way to specify a function signature of the Callable typing object, since the function could include multiple arguments with different types.

Reply:
Oh right forget my comment ;)